es6-operators/main.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-unused-vars */
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+
+// 1st solution
+// if(v1){
+//   console.log(v1)
+// }
+
+// if(v2){
+//   console.log(v2)
+// }
+
+v1 && console.log('v1 is truthy');
+
+// for v2
+v2 && console.log('v2 is truthy');
+
+/* TODO:
+ * Demonstrate how to assign a "default value" to a variable.
+ * In one expression, assign `'default-value'` to the variable `config1` if v1 is falsy.
+ * In one expression, assign `'default-value'` to the variable `config2` if v2 is falsy.
+ * Log both `config1` and `config2`.
+ * Expected: `config1 = { value: 'Something' }`, `config2 = 'default-value'`.
+ */
+
+// 2nd solution
+
+const config1 = undefined || console.log(v1);
+
+const config2 = undefined || console.log(v2);
+
+//
+
+/* TODO:
+ * Demonstrate how to assign a "default value" to a variable only if the variable is `undefined`.
+ * In one expression, assign `'default-value'` to the variable `cfg1` if v1 is undefined.
+ * In one expression, assign `'default-value'` to the variable `cfg2` if v2 is undefined.
+ * In one expression, assign `'default-value'` to the variable `cfg3` if v3 is undefined.
+ * Log `cfg1`, `cfg2`, and `cfg3`.
+ * Expected: `config1 = { value: 'Something' }`, `config2 = ''`, `config3 = 'default-value'`.
+ */
+
+// 3rd solution
+
+const cfg1 = v1 ?? console.log('default value');
+const cfg2 = v2 ?? console.log('default value');
+const cfg3 = v3 ?? console.log('default value');
+
+/* TODO:
+ * Use the shorthand if/else, which is especially useful for variable assignments.
+ * In one expression, assign `truthy` to the variable `tern1` if v1 is truthy
+ * and assign `falsy` to the same variable if v1 is falsy.
+ * Do the same for the variable `tern2` and the variable v2.
+ * Log `tern1` and `tern2`.
+ * Expected: `tern1 = 'truthy'`, `tern2 = 'falsy'`.
+ */
+
+// 4th solution
+
+let tern1;
+
+v1 ? (tern1 = console.log(true)) : console.log(undefined);
+
+!v1 ? (tern1 = console.log(false)) : console.log(undefined);
+
+v2 ? (tern1 = console.log(true)) : console.log(undefined);
+
+!v2 ? (tern1 = console.log(false)) : console.log(undefined);
```

Comment:
You should only have 2 lines of code for your variables `tern1` & `tern2`, and your console log will come afterwards